snapshot/policy/error_handling_policy.go

```diff
@@ -0,0 +1,33 @@
+package policy
+
+// ErrorHandlingPolicy describes files to be ignored when taking snapshots.
+type ErrorHandlingPolicy struct {
+	IgnoreFileErrors         bool `json:"ignoreFileErrs,omitempty"`
+	IgnoreFileErrorsSet      bool `json:"ignoreFileErrsSet,omitempty"`
+	NoParentIgnoreFileErrors bool `json:"noParentIgnoreFileErrs,omitempty"`
+
+	IgnoreDirectoryErrors         bool `json:"ignoreDirErrs,omitempty"`
+	IgnoreDirectoryErrorsSet      bool `json:"ignoreDirErrsSet,omitempty"`
+	NoParentIgnoreDirectoryErrors bool `json:"noParentIgnoreDirErrs,omitempty"`
+}
+
+// Merge applies default values from the provided policy.
+func (p *ErrorHandlingPolicy) Merge(src ErrorHandlingPolicy) {
```

Comment:
It'd be good to have unit tests for this and the `cli.setErrorHandlingPolicyFromFlags` function.

Reply:
done and done